tests/integrated/test_suite

```diff
@@ -100,42 +206,80 @@ test_type = "Making" if args.make else "Running"
 
 print("======= {} {} {} tests ========".format(test_type, len(tests), savepath.split("/")[-1]))
 
+avail = num_jobs
```

Comment:
What is `avail`? Name suggests something is "available", but here it's number of jobs and later it is modified by the job cost, which feel like very different things

Reply:
well, there are jobs and jobs - jobs in the terms of the make system, where a job can use a core, and jobs in terms of tests, where a test can use several threads. Will have to come up with different names :+1: 

Is `avail_threads` better? I am trying to use threads for the core-count, and tests for the number of tests ...